BullsAndCows/BullsAndCowsGame.cs

```diff
@@ -1,20 +1,48 @@
 ﻿using System;
+using System.Linq;
 
 namespace BullsAndCows
 {
     public class BullsAndCowsGame
     {
         private readonly SecretGenerator secretGenerator;
+        private readonly string secret = string.Empty;
         public BullsAndCowsGame(SecretGenerator secretGenerator)
         {
             this.secretGenerator = secretGenerator;
+            this.secret = this.secretGenerator.GenerateSecret();
         }
 
         public bool CanContinue => true;
```

Comment:
question: how can you stop game if you don't change "CanContinue" value?

Reply:
I break the loop in the man function